ibtk/include/ibtk/SecondaryHierarchy.h

```diff
@@ -148,6 +137,12 @@ struct SecondaryHierarchy
      */
     int d_finest_patch_level_number;
 
+    /**
+     * Tag strategy. Always does nothing since this class only repartitions and
```

Comment:
Can we omit it?

Reply:
I should reword this comment - it doesn't really do nothing, it just tags cells to exactly duplicate the tagging done to create the primary hierarchy.

Reply:
Fixed!